src/main/java/com/alipay/oceanbase/rpc/protocol/payload/impl/ObTableObjType.java

```diff
@@ -0,0 +1,285 @@
+package com.alipay.oceanbase.rpc.protocol.payload.impl;
+
+import com.alipay.oceanbase.rpc.util.Serialization;
+import io.netty.buffer.ByteBuf;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum ObTableObjType {
```

Comment:
Maybe ObTableObjct is a better name. Or you could leave a comment here

Reply:
just consistent with server